tests/integration/conftest.py

```diff
@@ -438,34 +466,38 @@ def _load_json_schema(path):
 
 
 @pytest.fixture
-def pre_registered(shell, request):
+def pre_registered(shell, request, yum_conf_exclude):
     """
     A fixture to install subscription manager and pre-register the system prior to the convert2rhel run.
+    For Oracle Linux we're using the _add_client_tools_repo_oracle to enable the client-tools repository
+    to install the subscription-manager package from.
+    We also exclude the rhn-client* packages for the same reason.
+    On Oracle Linux subscription-manger is obsolete and replaced by rhn-client* packages when installing subman.
     """
     username = TEST_VARS["RHSM_USERNAME"]
     password = TEST_VARS["RHSM_PASSWORD"]
     # Use custom parameters when the fixture is parametrized
     if hasattr(request, "param"):
         username, password = request.param
+        print(">>> Using username and password requested in the fixture parameter.")
+
+    # Add the client-tools repository for Oracle linux to install subscription-manager from
+    # Exclude the rhn-client* packages in the yum.conf
     if "oracle" in SYSTEM_RELEASE_ENV:
-        # Documentation does not contain steps on how to subscribe OL systems
-        # https://issues.redhat.com/browse/RHELC-1485
-        pytest.fail(
-            "Subscription manager is not present in Oracle Linux repositories. Please refer to RHELC-1485 for more"
-        )
+        yum_conf_exclude(exclude=["rhn-client*"])
+        _add_client_tools_repo_oracle(shell)
 
     assert shell("yum install -y subscription-manager").returncode == 0
     # Download the SSL certificate
     shell(
-        "curl --create-dirs -o /etc/rhsm/ca/redhat-uep.pem https://cdn-public.redhat.com/content/public/repofiles/redhat-uep.pem"
+        "curl --create-dirs -ko /etc/rhsm/ca/redhat-uep.pem https://cdn-public.redhat.com/content/public/repofiles/redhat-uep.pem"
     )
```

Comment:
I would update the above comment to include that we are downloading the certificate allowing insecure connection to the server (because we are using `-k` option)